### PR TITLE
complimentary error function

### PR DESCRIPTION
**Pull request by RiteshKB**

### CODE REVIEW

**Comment by certik** [comment]
Hi @RiteshKB thanks for the patch. Can you add some tests for the erfc function as well please? Otherwise I think it looks good.

File: symengine/functions.h [hunk 1]
```diff
@@ -924,6 +924,30 @@ class Erf : public OneArgFunction
 //! Canonicalize Erf:
 RCP<const Basic> erf(const RCP<const Basic> &arg);
 
+class Erfc : public OneArgFunction
+{
+    /*   The Gauss error function. This function is defined as:
+     *
+     *   .. math::
+     *      \mathrm{erfc}(x) =1 neg( \frac{2}{\sqrt{\pi}} \int_0^x e^{-t^2})
```
Comment (certik) [inline symengine/functions.h]: Should `neg` be repleaced with `-`?

Also, the parentheses should probably be removed. Like this:

```
\mathrm{erfc}(x) = 1 - \frac{2}{\sqrt{\pi}} \int_0^x e^{-t^2}
```

File: symengine/functions.h [hunk 2]
```diff
@@ -924,6 +924,30 @@ class Erf : public OneArgFunction
 //! Canonicalize Erf:
 RCP<const Basic> erf(const RCP<const Basic> &arg);
 
+class Erfc : public OneArgFunction
+{
+    /*   The Gauss error function. This function is defined as:
```
Comment (certik) [inline symengine/functions.h]: Isn't this the complementary error function?

Comment (RiteshKB) [inline symengine/functions.h]: yes @certik

**Comment by RiteshKB** [comment]
Im working on tests of erfc function.

File: nano.save [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (certik) [inline nano.save]: You should remove this file.

File: symengine/functions.h [hunk 4]
```diff
@@ -924,6 +924,30 @@ class Erf : public OneArgFunction
 //! Canonicalize Erf:
 RCP<const Basic> erf(const RCP<const Basic> &arg);
 
+class Erfc : public OneArgFunction
+{
+    /*   The complementary error function. This function is defined as:
+     *
+     *   .. math::
+     *      \mathrm{erfc}(x) =1 - \frac{2}{\sqrt{\pi}} \int_0^x e^{-t^2}
+     *\mathrm{d}t.
```
Comment (certik) [inline symengine/functions.h]: I would put a space after `=`, and also indent the second line with `\mathrm{d}t.` to the right, something like this:
```
*                \mathrm{d}t.
```

**Pull request closed without merging**